src/darwin/Framework/CHIP/CHIPAttestationTrustStoreBridge.mm

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import "CHIPAttestationTrustStoreBridge.h"
+
+chip::ByteSpan asByteSpan(NSData * value) { return chip::ByteSpan(static_cast<const uint8_t *>(value.bytes), value.length); }
+
+void CHIPAttestationTrustStoreBridge::Init(NSArray<NSData *> * paaCerts) { mPaaCerts = paaCerts; }
+
+CHIP_ERROR CHIPAttestationTrustStoreBridge::GetProductAttestationAuthorityCert(
+    const chip::ByteSpan & skid, chip::MutableByteSpan & outPaaDerBuffer) const
+{
+    VerifyOrReturnError(!skid.empty() && (skid.data() != nullptr), CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
```suggestion
    VerifyOrReturnError(!skid.empty(), CHIP_ERROR_INVALID_ARGUMENT);
```
(having a null data when `empty()` returns false is not a valid state for a Span).